patchwork-extensions-blockentity/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "patchwork-extensions-blockentity"
+version = getSubprojectVersion(project, "0.1.0")
+
+dependencies {
+	compile project(path: ':patchwork-capabilities', configuration: 'dev')
```

Comment:
This dependency is needed because:
```
public interface IForgeTileEntity extends ICapabilitySerializable<CompoundTag>
```
However, it might be possible to remove it by moving some of the default impls from IForgeTileEntity to the capability package. I would like to hear other's suggestion on this.

Reply:
This is fine how it is IMO, if you're going to need IForgeTileEntity you're going to need capabilities. Not adding this dependency would just make things confusing